src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -121,7 +121,7 @@ void RaftReplDev::async_alloc_write(sisl::blob const& header, sisl::blob const&
         // Write the data
         data_service().async_write(rreq->value, rreq->local_blkid).thenValue([this, rreq](auto&& err) {
             if (!err) {
-                auto raft_status = m_state_machine->propose_to_raft(std::move(rreq));
+                auto raft_status = m_state_machine->propose_to_raft(rreq);
```

Comment:
to overcome the inc/dec, is there a reason why this lambda can't be declared as `mutable` so that rreq can be non-const?